integration_tests/projects/001_flow_run_with_selectors/data/time_series.csv

```diff
@@ -1,4 +1,4 @@
-id,y,ds
+,y,ds
```

Comment:
Why this change?

Reply:
Already been merged, there was a conflict